cp3pt0-deployment/setup_tenant.sh

```diff
@@ -523,8 +523,17 @@ EOF
     
         # Check if the patch was successful
         if [[ $? -eq 0 ]]; then
-            success "Successfully patched CommonService CR in ${OPERATOR_NS}\n"
-            break
+            checkOperatorNS=$(${OC} get commonservice common-service -n ${OPERATOR_NS} -o yaml | yq '.spec.operatorNamespace=="'${OPERATOR_NS}'"')
+            checkServicesNS=$(${OC} get commonservice common-service -n ${OPERATOR_NS} -o yaml | yq '.spec.servicesNamespace=="'${SERVICES_NS}'"')
+            if [ $checkOperatorNS ] && [ $checkServicesNS ]; then
+                success "Successfully patched CommonService CR in ${OPERATOR_NS}"
+                break
+            else
+                operatorNSinCR=$(${OC} get commonservice common-service -n ${OPERATOR_NS} -o yaml | yq '.spec.operatorNamespace')
+                servicesNSinCR=$(${OC} get commonservice common-service -n ${OPERATOR_NS} -o yaml | yq '.spec.servicesNamespace')
+                info "OperatorNamespace in CommonService cr should be ${OPERATOR_NS}, but it is ${operatorNSinCR}"
+                info "ServicesNamespace in CommonService cr should be ${SERVICES_NS}, but it is ${servicesNSinCR}"
```

Comment:
I am wondering that should we raise `warning` msg here, and subtract one retry times.

As a result, it will retry several times, and error out the script when retry is zero.